flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/api/PlannerConfigOptions.java

```diff
@@ -52,95 +54,117 @@
 	public static final ConfigOption<Boolean> SQL_OPTIMIZER_SHUFFLE_PARTIAL_KEY_ENABLED =
 			key("sql.optimizer.shuffle.partial-key.enabled")
 					.defaultValue(false)
-					.withDescription("Enables shuffle by partial partition keys. " +
+					.withDescription("Enables shuffling by partial partition keys. " +
 							"For example, A join with join condition: L.c1 = R.c1 and L.c2 = R.c2. " +
 							"If this flag is enabled, there are 3 shuffle strategy:\n " +
 							"1. L and R shuffle by c1 \n" +
 							"2. L and R shuffle by c2\n" +
 							"3. L and R shuffle by c1 and c2\n" +
 							"It can reduce some shuffle cost someTimes.");
 
+	@Documentation.ExcludeFromDocumentation(value = "Temporary solution to enable optimization that removes redundant sort for SortMergeJoin, " +
+			"and this config option will be removed later.")
 	public static final ConfigOption<Boolean> SQL_OPTIMIZER_SMJ_REMOVE_SORT_ENABLED =
-			key("sql.optimizer.smj.remove-sort.enabled")
+			key("sql.optimizer.sortmergejoin.remove-sort.enabled")
```

Comment:
I suggest to use `sql.optimizer.sort-merge-join.remove-sort.enabled` to keep align style with other options.

Reply:
Thanks, fixed it.